src/server/utils/settings-store/settings-store.ts

```diff
@@ -15,13 +15,13 @@
  * limitations under the License.
  */
 
-import { AppSettings } from "../../../common/models/app-settings/app-settings";
+import { ServerAppSettings } from "../../../common/models/app-settings/app-settings";
 import { Nullary } from "../../../common/utils/functional/functional";
 
-type SettingsPromise = Nullary<Promise<AppSettings>>;
+type SettingsPromise = Nullary<Promise<ServerAppSettings>>;
 
 export class SettingsStore {
```

Comment:
What is it? Do we need this class?

Reply:
We provide settings as a promise to backend services. This class is probably overenginered wrapper for typed Promise. I will look into it and fix or promote to different task.

Reply:
Aaaand, done!